codeforamerica/vita-min
# GYR1-598 Add + use XFA-less 13614-C PDF for TY2024

## Link to JIRA issue
- https://codeforamerica.atlassian.net/browse/GYR1-598

## Is PM acceptance required?
- No - merge after code review approval

**Reminder**: merge main into this branch and get green tests before merging to main

## What was done?
- Add XFA-less 13614-C PDF for TY2024
- Update app/lib/pdf_filler/f13614c_pdf.rb to point to the new PDF
- Set a few spec tests to pending. They are commented with `# TODO reenable for TY2024`. As we move through successive tickets, we'll reenable them. 

## How to test?

- Ensure the checked-in PDF works with the 'Prepare a form' tool in Adobe Acrobat Pro (see my comment on the Jira ticket for details).
- Beyond that, not much to test. This PR sets us up for other tickets we're going to work on.
